userops_ext.go

```diff
@@ -595,3 +911,58 @@ func (op *UserOperation) String() string {
 		formatBytes(op.Signature),
 	)
 }
+
+// encodeCrossChainCallData encodes the call data in the cross-chain format
+func (op *UserOperation) encodeCrossChainCallData(intentJSONBytes []byte) ([]byte, error) {
```

Comment:
it's only used by the tests. Why not move it to the test file as a mocking function?